app/src/models/DataSourceI18n.ts

```diff
@@ -0,0 +1,929 @@
+          field: "geographical_location",
+        },
+        startYear: {
+          type: DataTypes.INTEGER,
+          allowNull: true,
+          field: "start_year",
+        },
+        endYear: {
+          type: DataTypes.INTEGER,
+          allowNull: true,
+          field: "end_year",
+        },
+        latestAccountingYear: {
+          type: DataTypes.INTEGER,
+          allowNull: true,
+          field: "latest_accounting_year",
+        },
+        frequencyOfUpdate: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+          field: "frequency_of_update",
+        },
+        spatialResolution: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+          field: "spatial_resolution",
+        },
+        language: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        accessibility: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        dataQuality: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+          field: "data_quality",
+        },
+        notes: {
+          type: DataTypes.TEXT,
+          allowNull: true,
+        },
+        units: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        methodologyUrl: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+          field: "methodology_url",
+        },
+        methodologyDescription: {
+          type: DataTypes.TEXT,
+          allowNull: true,
+          field: "methodology_description",
+        },
+        transformationDescription: {
+          type: DataTypes.TEXT,
+          allowNull: true,
+          field: "transformation_description",
+        },
+        publisherId: {
+          type: DataTypes.UUID,
+          allowNull: true,
+          references: {
+            model: "Publisher",
+            key: "publisher_id",
+          },
+          field: "publisher_id",
+        },
+        retrievalMethod: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+          field: "retrieval_method",
+        },
+        apiEndpoint: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+          field: "api_endpoint",
+        },
+        sectorId: {
+          type: DataTypes.UUID,
+          allowNull: true,
+          references: {
+            model: "Sector",
+            key: "sector_id",
+          },
+          field: "sector_id",
+        },
+        subsectorId: {
+          type: DataTypes.UUID,
+          allowNull: true,
+          references: {
+            model: "SubSector",
+            key: "subsector_id",
+          },
+          field: "subsector_id",
+        },
+        subcategoryId: {
+          type: DataTypes.UUID,
+          allowNull: true,
+          references: {
+            model: "SubCategory",
+            key: "subcategory_id",
+          },
+          field: "subcategory_id",
+        },
+      },
```

Comment:
Need to have `DataTypes.JSONB` column `type` here for the relevant columns

Reply:
Also using TEXT over STRING(255) as mentioned on the migration

Reply:
> Need to have `DataTypes.JSONB` column `type` here for the relevant columns

Is there somewhere I'm missing? the only fields that come translated are
transformation_description, dataset_description, methodology_description, dataset_name

Reply:
Then not, I suppose it's mostly proper names, which usually don't need to be translated.